src/autoimport/services.py

```diff
@@ -49,7 +49,7 @@ def fix_files(files: Tuple[TextIOWrapper]) -> Optional[str]:
     return None
 
 
-def fix_code(original_source_code: str) -> str:
+def fix_code(original_source_code: str, config: Optional[Dict[str, Any]] = None) -> str:
```

Comment:
Why did you set this `config` type as `Optional`? If it was to keep the interface in case anyone was already using `autoimport` as a library (which I doubt :P), then shouldn't `fix_files` argument should also be `Optional`?

Reply:
You're right, I should make `fix_files` consistent with `fix_code`.

I defined the signature with a keyword argument `def fix_code(..., config: Optional[...] = None)` so that the existing tests would not need to be updated (as there are many tests that call the `fix_code` function without passing a `config` argument).

Reply:
Updated in https://github.com/lyz-code/autoimport/pull/160/commits/9ff98d4cc752c53dce9c61a90e1594635d6e34c9.

Reply:
Makes sense also to keep backwards compatibility, thanks!